api/ring-camera.ts

```diff
@@ -46,6 +47,19 @@ function getBatteryLevel(data: CameraData) {
   return batteryLevel
 }
 
+function getExistingDoorbellType(data: CameraData): DoorbellType | undefined {
+  if (
+    !data ||
+    !data.settings ||
```

Comment:
From what I see in my devices, it looks like `data` and `data.settings` should always be defined.  Did you see any cases where they might not be?

Reply:
Nope—I have two cameras, and both have `data.settings`. I’ll remove the unnecessary checks, thanks!

Reply:
Fixed in https://github.com/dgreif/ring/pull/152/commits/20324e41d9e21933e39a36c4bd152fdddb62fd8f